scripts/install-open5gs.sh

```diff
@@ -8,14 +10,19 @@ fi
 
 PS4='[DEBUG] '
 set -x
-
+# \
+# **Install Docker and needed libraries:**
+sudo apt-get install -y cmake make gcc g++ pkg-config libfftw3-dev libmbedtls-dev libsctp-dev libyaml-cpp-dev libgtest-dev
+sudo apt install docker-compose	
+sudo apt install docker.io	
+# \
 # Add Docker's official GPG key:
```

Comment:
Does this script come from elsewhere? Guessing https://docs.docker.com/engine/install/ubuntu/. If so, cite.